build/run

```diff
@@ -120,14 +130,19 @@ if [ -z "${DISABLE_NESTED_DOCKER}" ]; then
     -v ${HOME}/.docker/config.json:${BUILDER_HOME}/.docker/config.json \
     -v /var/run/docker.sock:/var/run/docker.sock"
 fi
+DOCKER_API=''
+
+if (( $(bc <<< "$DOCKER_SERVER_API < $DOCKER_CLIENT_API") )); then
+    DOCKER_API="-e DOCKER_API_VERSION=${DOCKER_SERVER_API}"
+fi
```

Comment:
is this really needed? What happens if we unconditionally pass DOCKER_API_VERSION as an env to the container?